go/control/api/api.go

```diff
@@ -124,6 +124,17 @@ type IdentityStatus struct {
 
 // RegistrationStatus is the node registration status.
 type RegistrationStatus struct {
+	// LastAttemptSuccessful is true if the last registration attempt has been
+	// successful.
+	LastAttemptSuccessful bool `json:"last_attempt_successful"`
+
+	// LastAttemptErrorMessage contains the error message if the last
+	// registration attempt has not been successful.
+	LastAttemptErrorMessage string `json:"last_attempt_error_message,omitempty"`
+
+	// LastAttempt is the time of the last registration attempt.
```

Comment:
```suggestion
	// LastAttempt is the time of the last registration attempt. In case the node has not attempted to register yet, it will be the zero timestamp.
```